Documentation/Modularization-concept.md

```diff
@@ -1,154 +1,103 @@
-- **Individual**: Parameter set describing the physiological properties of an individual. The parameter set referred to is limited to the parameters provided by PK-Sim (and, if extension of the SS require, some additional parameters).
-- **MoBi-project**: A MoBi-file (e.g. *.mbp3) containing modules, model configurations, models, observed data, etc.
-- [**Initial Conditions (IC) BB**](BuildingBlocks/InitialConditions-BB.md): formerly known as Molecular Start Values.
-- [**Parameter Values (PV) BB**](BuildingBlocks/ParameterValues-BB.md): formerly known as Parameter Start Values.
+  - _PK-Sim module_: A module based on a PK-Sim PBPK model. As best practice, a PK-Sim module should not be modified. Instead, all changes/extensions to the model should be done in extension modules. A PK-Sim module is converted into an extension module when edited by the user.
+  - _Extension module_: (xModule) Editable modules that contain any changes to the model structure made by the user.
+
+- **PK-Sim snapshot**: part of the PK-Sim module allowing the re-creation of the PK-Sim module.
+- **Model**: A combination of modules.
+- **Model structure**: Structural definition of the model, including the containers, connections, species, and active and passive processes, but excluding the parametrization and initial conditions of the final simulation.
+- **Simulation**: Combination of the **model** and the results of a simulation of this model. *Note:* The distinction between a model and a simulation is not obvious in the OSPS, and these terms are interchangeable to a greater extent.
+- **MoBi-project**: A MoBi-file (e.g. *.mbp3) containing modules, models, observed data, etc.
+- [*Initial Conditions (IC) BB*](BuildingBlocks/InitialConditions-BB.md): formerly known as Molecule Start Values.
+- [*Parameter Values (PV) BB*](BuildingBlocks/ParameterValues-BB.md): formerly known as Parameter Start Values.
+- **Individual**: Parameter set describing the physiological properties of an individual. The parameter set referred to is limited to the parameters provided by PK-Sim. Technically comparable to the Parameter Values BB with additional metadata. See [*Individuals*](BuildingBlocks/Individuals-BB.md) for details.
+- **Expression profile**: Parameter set describing the expression of a protein. See [*Expression Profile*](BuildingBlocks/ExpressionProfile.md) for details.
 
 ## Project structure
 A MoBi-project contains a set of:
 - PK-Sim modules
 - Extension modules
 - Individual BBs
 - Expression Profile BBs
-- Model Configurations, which are combinations of (0-n) PK-Sim modules, (0-n) extension modules, (0-1) individual BB, and (0-n) expression profiles.
-- Models resp. simulations, which are instances of model configurations.
-
-![User interface of a MoBi project - expression profiles missing!](Figures/ProjectUI_Mockup.png)
+- Models resp. Simulations, which are combinations of (0-n) PK-Sim modules, (0-n) extension modules, (0-1) individual BB, and (0-n) expression profiles.
```

Comment:
Maybe add that at least 1 Module is required (otherwise it looks like I could create a simulation from 0 entities of each kind :))